tests/integration/test_vulnerability_detector/test_providers/test_multiple_provider_feeds.py

```diff
@@ -140,6 +141,9 @@ def test_check_log_multiple_provider_feeds(configuration, metadata, set_wazuh_co
         - 'Refresh of <provider_name> database finished'
         - 'The update of the <provider_name> feed finished'
     '''
+    # Restart Wazuh
+    control_service('restart')
```

Comment:
Is this necessary?

You are checking the comparability of other tests. Very careful with this. Discuss.

Reply:
This restart was added because other tests check an invalid configuration in `ossec.conf` so we need to restart the services after the new configuration is applied (in the fixture) and they don't revert the changes. This was a quick solution. The better solution is modify the previous test to rever the changes applied to the configuration

Reply:
Applied change and removed the control service restart line. Launched three Instances :green_circle: 
[Reports](https://github.com/wazuh/wazuh-qa/files/8813917/MultipleProviders.zip)